code/datums/gamemodes/nuclear_war.dm

```diff
@@ -37,6 +37,12 @@
 		/datum/job/xenomorph = NUCLEAR_WAR_LARVA_POINTS_NEEDED,
 	)
 
+	// Nuclear War Xeno Player Requirements to Evolve, applied with loop in post_setup().
```

Comment:
We generally don't make comments like these, no need. If someone wants to know what it does they can look at the initial definition.

Reply:
Comments adjusted as suggested.